ui/src/app/subnets/views/SubnetDetails/SubnetSummary/SubnetSummary.tsx

```diff
@@ -1,7 +1,233 @@
+                DNS
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-dns">{subnet.dns_servers}</p>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-description">
+                Description
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-description">{subnet.description}</p>
+            </Col>
+          </Row>
+        </Col>
+        <Col size={6}>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted">
+                <span id="subnet-managed-allocation">Managed allocation</span>{" "}
+                <Tooltip
+                  message="MAAS allocates IP addresses 
+from this subnet, exluding the 
+reserved and dynamic ranges."
+                  position="btm-right"
+                >
+                  <Icon name="information" />
+                </Tooltip>
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-managed-allocation">
+                {subnet.managed ? "Enabled" : "Disabled"}
+              </p>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-active-discovery">
+                <span id="subnet-active-discovery">Active discovery</span>{" "}
+                <Tooltip
+                  message="When enabled, MAAS will scan 
+this subnet to discover hosts 
+that have not been discovered 
+passively."
+                  position="btm-right"
+                >
+                  <Icon name="information" />
+                </Tooltip>
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-active-discovery">
+                {subnet.active_discovery ? "Enabled" : "Disabled"}
+              </p>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-proxy-access">
+                <span id="subnet-proxy-access">Proxy access</span>{" "}
+                <Tooltip
+                  message="MAAS will allow clients from 
+this subnet to access the 
+MAAS proxy."
+                  position="btm-right"
+                >
+                  <Icon name="information" />
+                </Tooltip>
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-proxy-access">
+                {subnet.allow_proxy ? "Allowed" : "Disallowed"}
+              </p>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-allow-dns-resolution">
+                <span id="subnet-allow-dns-resolution">
+                  Allow DNS resolution
+                </span>{" "}
+                <Tooltip
```

Comment:
It looks like this tooltip should only appear if `subnet.allow_dns === true` https://github.com/canonical-web-and-design/maas-ui/blob/master/legacy/src/app/partials/subnet-details.html#L436

There's also another tooltip for `subnet.allow_dns === false` https://github.com/canonical-web-and-design/maas-ui/blob/master/legacy/src/app/partials/subnet-details.html#L425